test/integration/acceptance/create_role_test.cpp

```diff
@@ -0,0 +1,223 @@
+      .sendTx(makeUserWithPerms(
+          {iroha::model::can_get_my_txs, iroha::model::can_create_role}))
+      .skipProposal()
+      .skipBlock()
+      .sendTx(completeTx(baseTx({iroha::model::can_get_my_txs}, "")));
+  ASSERT_ANY_THROW(itf.skipProposal());
+}
+
+/**
+ * @given some user with can_create_role permission
+ * @when execute tx with CreateRole command with empty permission
+ * @then the tx hasn't passed stateless validation
+ *       (aka skipProposal throws)
+ */
+TEST_F(CreateRole, EmptyPerms) {
+  IntegrationTestFramework itf;
+  itf.setInitialState(kAdminKeypair)
+      .sendTx(makeUserWithPerms(
+          {iroha::model::can_get_my_txs, iroha::model::can_create_role}))
+      .skipProposal()
+      .skipBlock()
+      .sendTx(completeTx(baseTx({}, kRole)));
+  ASSERT_ANY_THROW(itf.skipProposal());
+}
+
+/**
+ * @given some user with can_create_role permission
+ * @when execute tx with CreateRole command with too long role name
+ * @then the tx hasn't passed stateless validation
+ *       (aka skipProposal throws)
+ */
+TEST_F(CreateRole, LongRoleName) {
+  IntegrationTestFramework itf;
+  itf.setInitialState(kAdminKeypair)
+      .sendTx(makeUserWithPerms(
+          {iroha::model::can_get_my_txs, iroha::model::can_create_role}))
+      .skipProposal()
+      .skipBlock()
+      .sendTx(completeTx(
+          baseTx({iroha::model::can_get_my_txs}, std::string(8, 'a'))));
+  ASSERT_ANY_THROW(itf.skipProposal());
+}
+
+/**
+ * @given some user with can_create_role permission
+ * @when execute tx with CreateRole command with maximal role name size
+ * @then the tx is comitted
+ */
+TEST_F(CreateRole, MaxLenRoleName) {
+  IntegrationTestFramework()
+      .setInitialState(kAdminKeypair)
+      .sendTx(makeUserWithPerms(
+          {iroha::model::can_get_my_txs, iroha::model::can_create_role}))
+      .skipProposal()
+      .skipBlock()
+      .sendTx(completeTx(
+          baseTx({iroha::model::can_get_my_txs}, std::string(7, 'a'))))
+      .skipProposal()
+      .checkBlock(
+          [](auto &block) { ASSERT_EQ(block->transactions().size(), 1); })
+      .done();
+}
+
+/**
+ * @given some user with can_create_role permission
+ * @when execute tx with CreateRole command with inexistent permission name
+ * @then the tx hasn't passed stateless validation
+ *       (aka skipProposal throws)
+ */
+TEST_F(CreateRole, DISABLED_InexistentPerm) {
+  IntegrationTestFramework itf;
+  itf.setInitialState(kAdminKeypair)
+      .sendTx(makeUserWithPerms(
+          {iroha::model::can_get_my_txs, iroha::model::can_create_role}))
+      .skipProposal()
+      .skipBlock()
+      .sendTx(completeTx(baseTx({"this_permission_doesnt_exist"}, kRole)));
+  ASSERT_ANY_THROW(itf.skipProposal());
+}
+
+/**
+ * @given some user with can_create_role permission
+ * @when execute tx with CreateRole command with existent role name
+ * @then there is an empty verified proposal
+ */
+TEST_F(CreateRole, DISABLED_ExistingRole) {
```

Comment:
Please reflect actual status and disabling reason for C250 on [Acceptance Tests](https://soramitsu.atlassian.net/wiki/spaces/IS/pages/299958273/Acceptance+Tests) page.